<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Edit_Clientes</class>
 <widget class="QFrame" name="Form_Edit_Clientes">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LC INFORMÁTICA| EDITAR CLIENTE</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/clientes.png</normaloff>../../img/clientes.png</iconset>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="fr_FormClientes">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFF;
border: none</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0" colspan="3">
     <widget class="QLineEdit" name="tx_Nome">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
      </property>
      <property name="placeholderText">
       <string>NOME COMPLETO</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="lb_Descricao">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: bold;

border-bottom: 2px solid #A2A2A2
}</string>
      </property>
      <property name="text">
       <string>FICHA CADASTRAL CLIENTE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="lb_FormTelefone">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
      </property>
      <property name="text">
       <string>TELEFONE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="lb_FormCpf">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
      </property>
      <property name="text">
       <string>CPF</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLineEdit" name="tx_cpf">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; 
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
      </property>
      <property name="inputMask">
       <string>000.000.000-00</string>
      </property>
      <property name="text">
       <string>..-</string>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #CFCFCF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="lb_FormNome">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
      </property>
      <property name="text">
       <string>NOME</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="tx_Telefone">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; 
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
      </property>
      <property name="inputMask">
       <string>(00) 0000-00000</string>
      </property>
      <property name="text">
       <string>() -</string>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="cursorPosition">
       <number>15</number>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="lb_Linha">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: normal;

border-bottom: 2px solid #A2A2A2;
color: #797979
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="lb_DescrecaoEndereco">
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
      </property>
      <property name="text">
       <string>ENDEREÇO</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QLineEdit" name="tx_Endereco">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;

}
QLineEdit:Focus {
border: 1px solid red;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Endereço ou localidade</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="fr_Botoes">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>700</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);
border:none;

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <widget class="QPushButton" name="Bt_Salvar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>2</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string>SALVAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/salvar/salvar.png</normaloff>:/salvar/salvar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_Voltar">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>2</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/sair/delete_delete_exit_1577.png</normaloff>:/sair/delete_delete_exit_1577.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tx_Nome</tabstop>
  <tabstop>tx_cpf</tabstop>
  <tabstop>tx_Telefone</tabstop>
  <tabstop>tx_Endereco</tabstop>
 </tabstops>
 <resources>
  <include location="../Icones/icones.qrc"/>
 </resources>
 <connections/>
</ui>
